File: 1100/519B/main.java
import java.util.*;
 
public class Main 
{
	public static void main(String[] args) 
	{
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		
		long sumA = 0, sumB = 0, sumC = 0;
		
		for (int i = 0; i < n; i++) 
		    sumA += sc.nextLong();
		for (int i = 0; i < n - 1; i++) 
		    sumB += sc.nextLong();
		    
		System.out.println(sumA - sumB);
		
		for (int i = 0; i < n - 2; i++) 
		    sumC += sc.nextLong();
		    
		System.out.println(sumB - sumC);
	}
}

File: 1100/519B/main.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;

    long long sumA = 0, sumB = 0, sumC = 0;

    // Read n numbers
    for (int i = 0; i < n; i++) 
    {
        long long x;
        cin >> x;
        sumA += x;
    }

    // Read n-1 numbers
    for (int i = 0; i < n - 1; i++) 
    {
        long long x;
        cin >> x;
        sumB += x;
    }

    cout << (sumA - sumB) << endl;

    // Read n-2 numbers
    for (int i = 0; i < n - 2; i++) 
    {
        long long x;
        cin >> x;
        sumC += x;
    }

    cout << (sumB - sumC) << endl;

    return 0;
}
